vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmations.cc

```diff
@@ -240,11 +240,36 @@ void Confirmations::OnDidSendConfirmation(
               << confirmation.transaction_id << " and creative instance id "
               << confirmation.creative_instance_id);
 
+  if (delegate_) {
+    delegate_->OnDidConfirm(confirmation);
+  }
+
   StopRetrying();
 
   ProcessRetryQueue();
 }
 
+void Confirmations::OnFailedToSendConfirmation(
```

Comment:
Eventually we will decouple sending confirmations and redeeming unblinded tokens as they have totally different code paths.